### PR TITLE
Update names to extension-point and extensionRegistry

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@benjaminapetersen here are the rename changes, PTAL

File: README.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ promise, data, etc.
 // args is an object set via the directive in the view.
 // likely it is some object on a controller scope that gives
 // meaning to the endpoint.
-extensionInput.register('endpoint1', function(args) {
+extensionRegistry.register('endpoint1', function(args) {
```
Comment (benjaminapetersen) [inline README.md:125]: Shall we now also update this to `extensionRegistry.add()`?  Seems redundant here lil bit.

File: README.md [hunk 2]
```diff
@@ -185,12 +185,12 @@ For organizational purposes, register endpoints ahead of time:
 
 ```javascript
 
-extensionInput.register('sidebar-left');
-extensionInput.register('main');
-extensionInput.register('footer');
-extensionInput.register('foo');
-extensionInput.register('bar');
-extensionInput.register('shizzle');
+extensionRegistry.register('sidebar-left');
```
Comment (benjaminapetersen) [inline README.md:188]: Yeah, def feels redundant with "Registry.register" :smile:

**Comment by benjaminapetersen** [comment]
Other than the .add() vs .register() looks good to me!

File: README.md (hunk 2 above)
Comment (jwforres) [inline README.md:188]: if we make it extensionRegistry.add it should probably be
extensionRegistry.remove

Comment (benjaminapetersen) [inline README.md:188]: Makes sense!

**Comment by benjaminapetersen** [comment]
ping!  just to keep it fresh :smile:

**Pull request merged**